### DISCUSSION TITLE
How to change the highlight color around text?

### DISCUSSION THREAD

**Discussion by HynDuf** [RafaelGB/obsidian-db-folder#831] [Q&A] [answered]
As you can see below, the highlight color of the text make it impossible to see in my dark mode?

![image](https://user-images.githubusercontent.com/29995756/219986709-f99fd0b0-55ce-4afd-933a-d8dbb7e02b6b.png)

Could you guys show me the CSS elements of these? Thanks in advance.

**Comment by RafaelGB** [comment, marked as answer]
Will be fixed with the next release

**Reply by RafaelGB** [reply]
https://github.com/RafaelGB/obsidian-db-folder/issues/809
